layouts/community/ergodox/azurefog/keymap.c

```diff
@@ -0,0 +1,385 @@
+#include "ergodox_ez.h"
+#include "debug.h"
```

Comment:
Could you remove all of these additional includes?  

```suggestion
```

They get included automatically, at various points in the built process, so they are redundant and not necessary here.